### ISSUE TITLE
Dashboard links fail to work after the upgrade to v. 8.4.11

### ISSUE THREAD

**Issue by heyrod** [grafana/grafana#59110]
We recently upgraded our Grafana instance from v. 7.1.5 to 8.4.11, which is latest compatible version for our old Elastic data sources. We've been using Grafana for nearly 6 years and have about two dozen organizations. In all Grafana organizations we use many old Table panels, which we cannot migrate to the new plugin version because the recommended solution (through the Migrate button) is breaking data structure in the panels.

So, after the upgrade, we noticed that the links embedded in Tables and designed to open other dashboards with predefined variables are not working any longer.  A short investigation has showed the URLs in the format like _/dashboard/db/dashboard_name?orgId=x_ should be replaced by _/d/uid/dashboard_name?orgId=x_ to work as before.

Is this change documented? Can you point to specific release notes and provide some background info for the change?

I think it's rather a bug because the result is different from expected.

Thanks

**Comment by mellieA** [comment]
@heyrod it looks like that could be possibly related to this change: 

https://github.com/grafana/grafana/pull/35104

Documented in the release notes for 8.0 here: 

https://grafana.com/docs/grafana/latest/release-notes/release-notes-8-0-0/#breaking-changes 

However, as far back as 7.1.5 you can see the format of  `/d/uid/. . . ` has been used https://github.com/grafana/grafana/issues/25115#issuecomment-691900892 

The code has been migrated several times so the history was hard to follow, I looked, but can't point to an exact release where a change was made. It likely dates back quite a bit.

**Comment by heyrod** [comment]
Hi [mellieA](https://github.com/mellieA), it was working two ways before the upgrade: dashboard/db/ and /d/uid. I think the support for the old endpoint was deprecated in some release and then removed completely in Grafana 8.0.0
This change should be included in the upgrade notes for Grafana 8.
